test/Interop/Cxx/stdlib/std-module-interface.swift

```diff
@@ -3,13 +3,12 @@
 // RUN: %target-swift-ide-test -print-module -module-to-print=std.string -source-filename=x -enable-cxx-interop -tools-directory=%llvm_obj_root/bin -module-cache-path %t | %FileCheck %s  -check-prefix=CHECK-STRING
 
 // Clang driver on Windows doesn't support -stdlib=libc++
-// XFAIL: OS=windows-msvc
+// UNSUPPORTED: OS=windows-msvc
```

Comment:
These are basically synonymous but it would probably be stronger to just say `requires macos` as that's what we really mean. 

Not too important as we should be fixing this in the next week or two anyway.